io_scene_hubs/panels.py

```diff
@@ -46,6 +41,37 @@ class HubsMaterialPanel(Panel):
     def draw(self, context):
         draw_components_list(self, context)
 
+class HubsGLTFExportPanel(bpy.types.Panel):
+
+    bl_space_type = 'FILE_BROWSER'
+    bl_region_type = 'TOOL_PROPS'
+    bl_label = "Hubs Components"
+    bl_parent_id = "GLTF_PT_export_user_extensions"
+    bl_options = {'DEFAULT_CLOSED'}
+
+    @classmethod
+    def poll(cls, context):
+        sfile = context.space_data
+        operator = sfile.active_operator
+        return operator.bl_idname == "EXPORT_SCENE_OT_gltf"
+
+    def draw_header(self, context):
+        props = bpy.context.scene.HubsComponentsExtensionProperties
+        self.layout.prop(props, 'enabled', text="")
```

Comment:
So if you uncheck this, none of the Hubs components stuff runs? Nice!

Reply:
Yep, its just checked for in each of the hooks